### DISCUSSION TITLE
请教：群晖DS116应该下载哪个版本？

### DISCUSSION THREAD

**Discussion by cn0755** [jeessy2/ddns-go#694] [Q&A] [answered]
此前安装其他spk，用的都是含有“armada38x”的spk，请问，这个应该下载哪个？谢谢。。。。

**Comment by WaterLemons2k** [comment]
应该是 armv7。

参考：https://cn.marvell.com/products/infrastructure-processors/multi-core-processors/armada/armada-38x.html

**Comment by cn0755** [comment]
非常感谢!

**Comment by cn0755** [comment]
小白还得再请教，怎么弄成spk让群晖安装？

**Reply by WaterLemons2k** [reply, marked as answer]
> 小白还得再请教，怎么弄成spk让群晖安装？

不需要打包成 spk。只需要连接到群晖的 SSH 或 Telnet 后按照 [系统中使用](https://github.com/jeessy2/ddns-go#系统中使用) 安装服务即可：

```sh
wget https://github.com/jeessy2/ddns-go/releases/download/v5.1.1/ddns-go_5.1.1_linux_armv7.tar.gz
tar xvzf ddns-go_5.1.1_linux_armv7.tar.gz ddns-go
./ddns-go -s install
```

**Reply by cn0755** [reply]
谢谢！

**Reply by cn0755** [reply]
已经成功安装，提示： 安装 ddns-go 服务成功! 请打开浏览器并进行配置。
但在套件中心看不到DDNS-GO的图标，怎么处理呢？
是不是远程（和nas不在一个局域网内）不能配置呢？





> > 小白还得再请教，怎么弄成spk让群晖安装？
> 
> 不需要打包成 spk。只需要连接到群晖的 SSH 或 Telnet 后按照 [系统中使用](https://github.com/jeessy2/ddns-go#%E7%B3%BB%E7%BB%9F%E4%B8%AD%E4%BD%BF%E7%94%A8) 安装服务即可：
> 
> ```shell
> wget https://github.com/jeessy2/ddns-go/releases/download/v5.1.1/ddns-go_5.1.1_linux_armv7.tar.gz
> tar xvzf ddns-go_5.1.1_linux_armv7.tar.gz ddns-go
> ./ddns-go -s install
> ```

**Reply by WaterLemons2k** [reply]
> 已经成功安装，提示： 安装 ddns-go 服务成功! 请打开浏览器并进行配置。
> 
> 但在套件中心看不到DDNS-GO的图标，怎么处理呢？
> 
> 是不是远程（和nas不在一个局域网内）不能配置呢？

安装后并不会在套件中心中显示图标。只需要在浏览器中打开群晖 ddns-go 的端口（默认 9876）就可以了。

假设群晖 IP 为 10.1.1.1，ddns-go 端口保持默认，则只需要打开 `10.1.1.1:9876` 就可以了。

**Reply by cn0755** [reply]
> > 已经成功安装，提示： 安装 ddns-go 服务成功! 请打开浏览器并进行配置。
> > 但在套件中心看不到DDNS-GO的图标，怎么处理呢？
> > 是不是远程（和nas不在一个局域网内）不能配置呢？
> 
> 安装后并不会在套件中心中显示图标。只需要在浏览器中打开群晖 ddns-go 的端口（默认 9876）就可以了。
> 
> 假设群晖 IP 为 10.1.1.1，ddns-go 端口保持默认，则只需要打开 `10.1.1.1:9876` 就可以了。

谢谢，这个我知道的。请教一下，如果要停止和卸载怎么办呢？

**Reply by WaterLemons2k** [reply]
> 谢谢，这个我知道的。请教一下，如果要停止和卸载怎么办呢？

停止：`sudo systemctl stop ddns-go`

卸载：`sudo ./ddns-go -s uninstall`

**Reply by cn0755** [reply]
> > 谢谢，这个我知道的。请教一下，如果要停止和卸载怎么办呢？
> 
> 停止：`sudo systemctl stop ddns-go`
> 
> 卸载：`sudo ./ddns-go -s uninstall`

非常非常感谢。
